### PR TITLE
getting-started folder content tune up

### PR DESCRIPTION
**Pull request by leslietilling**
## Purpose of this pull request

This pull request (PR) completes the clean up efforts for #816  and addresses the topics in the getting-started folder. Some headings were streamlined or clarified as needed, as well as the following items:

- Headings using sentence case and not infinitives
- Consistency in structure and style
- Edits for clarity

## Magento release version

- [ ] 2.4.x

   Specify a patch release number, if applicable:

- [x] 2.3.x

   Specify a patch release number, if applicable:

## Product editions

Is this update specific to a product edition?

- [ ] Magento Open Source only
- [ ] Magento Commerce only

Is this update specific to an installed feature extension

- [ ] B2B extension
- [ ] Other feature set, please specify:

## Affected documentation pages

https://docs.magento.com/user-guide/v2.3/getting-started/contribute.html
https://docs.magento.com/user-guide/v2.3/getting-started/login-credentials.html
https://docs.magento.com/user-guide/v2.3/getting-started/magento-ecosystem.html
https://docs.magento.com/user-guide/v2.3/getting-started/quick-tour.html
https://docs.magento.com/user-guide/v2.3/getting-started/resources.html

### CODE REVIEW

File: src/getting-started/magento-ecosystem.md [hunk 1]
```diff
@@ -1,35 +1,34 @@
...
 
-[Magento Forums][1]
-:  This is the place to find solutions, become acquainted with international communities, and join a special interest chat! Share your knowledge and earn kudos from others!
+Magento Forums
+:  The [forums][1]{:target="_blank"} are the place to find solutions, become acquainted with international communities, and join a special interest chat. Share your knowledge and earn kudos from others.
 
-[Facebook][2]{:target="_blank"}
-: Find out what’s happening and join the discussion on our Facebook page!
+Facebook
+: Find out what’s happening and join the discussion on our [Facebook page][2].
```
Comment (hickskenh) [inline src/getting-started/magento-ecosystem.md:11]:
```suggestion
: Find out what is happening and join the discussion on our [Facebook page][2].
```

File: src/getting-started/resources.md [hunk 2]
```diff
@@ -4,35 +4,17 @@ title: Resources
...
-
-[Partners][12]{:target="_blank"}
-:  Magento partners are committed to your success, and provide custom integrations, best-in-class customer experiences, strategic marketing initiatives, and expert performance and scalability optimization for both on-premise and cloud-based solutions.
-
-[Expert Consulting Services][13]{:target="_blank"}
-:  Our Expert Consulting Group (ECG) helps Magento merchants and Solution Partners maximize their success. Our experts offer comprehensive analysis and best practice recommendations, from architecture planning through post-deployment.
+[Product documentation][1]{:target="_blank"}|Whether you are a merchant, designer, [developer][2], or all of the above, this is where you can gain a deeper insight into how you can leverage the power of Magento to grow your business.|
+[Magento Resources Library][3]{:target="_blank"}|Gain access to expert insight and online business resources to help develop and improve your store. Ebooks, Magento whitepapers, and webinars.|
```
Comment (hickskenh) [inline src/getting-started/resources.md:8]: is store, ebooks, whitepapers all part of the same list?  if so, change there . after store.

File: src/getting-started/resources.md [hunk 3]
```diff
@@ -4,35 +4,17 @@ title: Resources
...
+[Events and webinars][8]{:target="_blank"}|Keep tabs on our calendar of upcoming events and webinars. There is always something happening in the Magento ecosystem.|
+[Magento Marketplace][9]{:target="_blank"}|Visit the Magento next-generation marketplace of digital commerce solutions. Magento Marketplace offers merchants a curated selection of solutions, while providing qualified developers the tools, platform, and prime location for a thriving business.|
+[Training and certification][10]{:target="_blank"}|Learn how to unleash the power and flexibility of Magento. We provide training for every role in your business, including marketers, designers, developers, and more. Experienced Magento professionals can validate their real-world skills by earning Magento certification.|
+[Partners][12]{:target="_blank"}|Magento partners are committed to your success, and provide custom integrations, best-in-class customer experiences, strategic marketing initiatives, and expert performance and scalability optimization for both on-premise and cloud-based solutions.|
```
Comment (hickskenh) [inline src/getting-started/resources.md:15]:
```suggestion
[Partners][12]{:target="_blank"}|Magento partners are committed to your success and provide custom integrations, best-in-class customer experiences, strategic marketing initiatives, and expert performance and scalability optimization for both on-premise and cloud-based solutions.|
```

**Review by hickskenh — APPROVED** [review]
approved after you review my comments/suggestions

**Comment by leslietilling** [comment]
running tests

**Pull request merged**